common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -3157,9 +3157,10 @@ def test_get_answer_with_jump_to_id_urls(self):
 
         data = {}
         problem = CapaFactory.create(showanswer='always', xml=problem_xml)
-        problem.runtime.replace_jump_to_id_urls = Mock()
-        problem.get_answer(data)
-        assert problem.runtime.replace_jump_to_id_urls.called
+        problem.runtime.service(problem, 'replace_urls').replace_urls = Mock()
+
+        answers = problem.get_answer(data)
```

Comment:
Nit: we're not checking these answers anywhere, so we don't need to assign them.

Reply:
done